package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "1.0.7",
+  "version": "1.1.0",
```

Comment:
This is fine for this PR, but going forward I would let `np` manage versioning after PR merge